api/src/main/java/org/openmrs/module/patientflags/metadatadeploy/handler/PriorityDeployHandler.java

```diff
@@ -0,0 +1,35 @@
+package org.openmrs.module.patientflags.metadatadeploy.handler;
+
+import org.openmrs.annotation.Handler;
+import org.openmrs.module.metadatadeploy.handler.AbstractObjectDeployHandler;
+import org.openmrs.module.patientflags.Priority;
+import org.openmrs.module.patientflags.api.FlagService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+@Handler(
+        supports = {Priority.class}
+)
+
+public class PriorityDeployHandler extends AbstractObjectDeployHandler<Priority> {
```

Comment:
It would be great to skip at least a line after the class declaration, before its members.